doc/langref.html.in

```diff
@@ -7147,8 +7149,8 @@ const namespace_name = @import("dir_name/file_name.zig");
 const TypeName = @import("dir_name/TypeName.zig");
 var global_var: i32 = undefined;
 const const_name = 42;
-const primitive_type_alias = f32;
-const string_alias = []u8;
+const PrimitiveTypeAlias = f32;
+const StringAlias = []u8;
```

Comment:
I don't think the term "string" is worth tacitly endorsing here, as Zig intentionally doesn't have a "string" type (see [test_slices.zig in this section](https://ziglang.org/documentation/master/#Slices)).

```suggestion
const SliceAlias = []u8;
```

or

```suggestion
```